.annotation_safe_list.yml

```diff
@@ -0,0 +1,10 @@
+auth.Group:
+  ".. no_pii::" : "No PII"
+auth.Permission:
+  ".. no_pii::" : "No PII"
+auth.User:
+  ".. pii::": "username and email address"
```

Comment:
For some reason, I didn't realize that if we don't do the tag prefix/suffix thing that bmez suggested a while back, then the safelist keys would need to use the fully marked up sphinx-type variant of the tag :(